corehq/tests/pytest_plugins/reusedb.py

```diff
@@ -75,8 +75,8 @@ def pytest_addoption(parser):
 
 @pytest.hookimpl
 def pytest_configure(config):
-    config.reuse_db = reusedb = config.getoption("--reusedb")
-    config.skip_setup_for_reuse_db = reusedb and reusedb != "reset"
+    config.reuse_db = reusedb = config.getoption("--reusedb") or config.getvalue("reuse_db")
+    config.skip_setup_for_reuse_db = reusedb and (reusedb != "reset" or config.getvalue("create_db"))
```

Comment:
The docs for `--create-db` say:

> Re-create the database, even if it exists. This option can be used to override --reuse-db.

It will not skip setup if `create_db` is true _unless_ reusedb is not false.

Another way to say that is `create_db` has no effect unless `reusedb` is true. Databases are always (re)created unless `reusedb` is true.